src/test/lib/HandshakeTest.cpp

```diff
@@ -885,13 +894,11 @@ QuicTestCustomServerCertificateValidation(
                 }
                 TEST_EQUAL(AcceptCert, Client.GetIsConnected());
 
-                if (AcceptCert) { // Server will be deleted on reject case, so can't validate.
```

Comment:
I don't see how the server would get deleted. The new connection instance is held by the unique_ptr above and the AutoDelete bit is not set, so the connection should be valid until the unique_ptr goes out of scope.

Reply:
I believe this was meant to handle a case where the listener could reject the connection, and then this pointer would never be set. That being said, I cannot see a path where that would happen in this test case (or the one below). And since the tests seem to pass just fine, I'm ok with these changes.